content/spin/kv-store.md

```diff
@@ -543,8 +543,8 @@ You can read the [improvement proposal for key/value support](https://github.com
 {
   "@context": "https://schema.org",
   "@type": "VideoObject",
-  "name": "Key/Value Store preview - Spin 0.9.0",
-  "description": "Overview of the key/value store in Spin v0.9.0",
+  "name": "Key/Value Store preview - Spin 0.10.0",
```

Comment:
Is this still a preview?  Is it specific to 0.10?  I'd suggest dropping the version from this point on.  Generally this page (with references to the improvement proposal etc. and discussion of motivation and future direction) feels like a bit "moment in time meet the new thing" when it maybe needs to shift to "there is a thing, here is how to use it."

But that may be a larger change than you want to make in this PR.

Reply:
Good point; the preview text was removed.